src/_lib/sass/src/_variables.scss

```diff
@@ -469,17 +469,28 @@ $state-danger-border:            get-color('tertiary', base) !default;
 //##
 
 //** Tooltip max width
-$tooltip-max-width:           300px !default;
+$tooltip-max-width:                300px !default;
 //** Tooltip text color
-$tooltip-color:               get-color("gray", base) !default;
+$tooltip-color:                    #FFF !default;
 //** Tooltip background color
-$tooltip-bg:                  #fff !default;
-$tooltip-opacity:             1 !default;
+$tooltip-bg:                        get-color("gray", x-light)!default;
+$tooltip-opacity:                   1 !default;
 
 //** Tooltip arrow width
-$tooltip-arrow-width:         5px !default;
+$tooltip-arrow-width:               5px !default;
 //** Tooltip arrow color
-$tooltip-arrow-color:         $tooltip-bg !default;
+$tooltip-arrow-color:               get-color("gray", x-light);
```

Comment:
Unless I'm missing something, this can be changed back to `$tooltip-bg !default`